config/api/routes/site.php

```diff
@@ -57,6 +57,10 @@
         ],
         'method'  => 'GET',
         'action'  => function () {
+            if ($this->pattern() === 'site/children/blueprints') {
+                deprecated('`GET site/children/blueprints` API endpoint has been deprecated and will be removed in 3.7.0. Use `GET site/blueprints');
```

Comment:
```suggestion
                deprecated('`GET site/children/blueprints` API endpoint has been deprecated and will be removed in 3.7.0. Use `GET site/blueprints` instead');
```

The `@todo` tag above should also be changed to 3.7.0. Not really critical, but better for our overview.